### PR TITLE
{math}[GCCcore/11.2.0] Gurobi v9.5.0 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3621.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/a06a3909be2bad856e02d533e4cf8b57 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2664.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/4a1c24946ffbaa9621d01f76490f4c9e for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/a760a33a2ea944307cd6299566a39245 for a full test report.

File: easybuild/easyconfigs/g/Gurobi/Gurobi-9.5.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+dependencies = [
+    ('Python', '3.9.6'),
+]
+
+# remove bundled Python interpreter in favour of the dependency in EB
+postinstallcmds = ['rm %(installdir)s/bin/python*']
+
+# license is not mandatory for installation
+license_file = HOME + '/licenses/%(name)s.lic'
```
Comment (verdurin) [inline easybuild/easyconfigs/g/Gurobi/Gurobi-9.5.0-GCCcore-11.2.0.eb:32]: @boegel this appears to be a bit misleading, given that the build fails without it.

**Review by verdurin — DISMISSED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
centos7.lan - Linux centos linux 7.9.2009, x86_64, Intel Core Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/4a6aeb3d0c676e604d428f538619b524 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3108.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/8f96c9330ffbdb4b2cfcd19004f40f05 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/e244b4cf0670d353570fc5016dd3fdf9 for a full test report.

File: easybuild/easyconfigs/g/Gurobi/Gurobi-9.5.0-GCCcore-11.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/g/Gurobi/Gurobi-9.5.0-GCCcore-11.2.0.eb:32]: @verdurin fixed in b3f588d

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**